CMakeLists.txt

```diff
@@ -347,7 +347,8 @@ find_package(SWIG)
 
 if(SWIG_FOUND)
     set(SWIG_VERSION_CHECK FALSE)
-    if("${SWIG_VERSION}" VERSION_GREATER ${GR_SWIG_MIN_VERSION})
+    if("${SWIG_VERSION}" VERSION_EQUAL ${GR_SWIG_MIN_VERSION} OR
+       "${SWIG_VERSION}" VERSION_GREATER ${GR_SWIG_MIN_VERSION})
```

Comment:
I think `GR_SWIG_MIN_VERSION` was adjusted to make the old code work. This is cleaner, but you'll also need to update the min version then.